app/models/aircraft.rb

```diff
@@ -15,6 +15,6 @@ def notify_user
   }
 
   def to_s
```

Comment:
I would prefer to use something like name or something here, not the to_s :)

Reply:
@hvillero the `to_s` will be used anytime the model is called in string interpolation. So for example in the email template doing `<%= @aircraft %>` will now automatically call the `to_s` method. It's a nice helper which should call `full_name` within it if anything.

Reply:
@pJeyakumar, disregard this note. Thx